### PR TITLE
updating readme

### PR DESCRIPTION
**Pull request by ananya**
Issue Number #84 
Changes proposed in this pull request :
[Live Link](provide link here)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,37 @@
...
+This project is exclusively build only for female contributors to give them a headstart in open source.
+Together we can make things better. OS4W aims to be a resource for connecting all women, including women of color and transgender women, to open source projects that are welcoming, inclusive, and appreciative of diversity in their contributors.
+# OUR GOALS
+This project is started with an aim to get more woman contributors to Open Source. What we want is more females enchanted with the mysteries of open source and to feel happy and proud writing codes or contributing to an open 
+# Contribute
+* To contribute to this project all you need is to be a part of opencode 18 and obviously a female developer. The best part then is you could find yourself as the contributor of this website.
+* Contributors are required to solve the issues and earn points.
+* At the end of competition, winner will be declared  on the basis of quality of issues they solved and the marks they got.
+# Prizes For Winner of Opencode
```
Comment (gauthamzz) [inline README.md]: Remove this

**Review by gauthamzz — CHANGES_REQUESTED** [review]
Don’t add details of opencode. Other than that this looks good. 

Also add opencode badge to readme.
Add the logo as well to the readme.

**Comment by ananya** [comment]
that means should delete everything except Girls who code, our goals, and scoring criterion? @gauthamzz

**Comment by gauthamzz** [comment]
WHy you need scoring criteria , only add things related to girls who code

**Comment by ananya** [comment]
@gauthamzz please review

**Review by Anupam-dagar — COMMENTED** [review]
why is one image added for readme update?

**Review by gauthamzz — APPROVED** [review]

**Pull request merged**